hydromt/data_adapter/dataframe.py

```diff
@@ -89,12 +89,25 @@ def __init__(
         name, catalog_name: str, optional
             Name of the dataset and catalog, optional for now.
         """
+        if driver_kwargs is None:
+            driver_kwargs = {}
+        if attrs is None:
+            attrs = {}
+        if meta is None:
+            meta = {}
+        if unit_add is None:
+            unit_add = {}
+        if unit_mult is None:
+            unit_mult = {}
+        if rename is None:
+            rename = {}
```

Comment:
Not sure this is "good practice", but can we not shorten this by setting eg`driver_kwargs = driver_kwargs or {}` when we pass the argument to the method (or in other places set a class attribute). Declaring all these variables first makes the script quite long. This happens at many places.

Reply:
correct. The two are equivalent, so the shorter version is equaly good practicise. I thought I'd use the short version everywhere, but apparently a few places slipped through the cracks. I'll update it!

Reply:
I've used the shorter version wherever I could find it. if you find other places where it could be applied, feel free to let me know